Library/Formula/cmake.rb

```diff
@@ -3,6 +3,7 @@ class Cmake < Formula
   homepage "https://www.cmake.org/"
   url "https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz"
   sha256 "b73f8c1029611df7ed81796bf5ca8ba0ef41c6761132340c73ffe42704f980fa"
+  revision 1
```

Comment:
I haven't been bumping `revision` for Linuxbrew. I keep the revision numbers in sync with Homebrew.